app/allocation/views/view.njk

```diff
@@ -123,6 +123,15 @@
           })}}
         </p>
       {% endif %}
+
+      {% if move.card.cancelAllocationHref %}
```

Comment:
With the suggestion in the controller you can then call these values directly on `move.` and not have to worry about the `move.card`.

The output on `move.card` is only to be passed directly into the card component so that it can be rendered.

You can bypass needing to do all of that.

Reply:
Good point.